pkgs/development/ocaml-modules/ocaml-protoc-plugin/default.nix

```diff
@@ -0,0 +1,46 @@
+{ lib, fetchFromGitHub, buildDunePackage, ocaml-ng, pkg-config, protobuf }:
+
+buildDunePackage rec {
+  pname = "ocaml-protoc-plugin";
+  version = "4.3.1";
+
+  duneVersion = "3";
+
+  minimalOcamlVersion = "4.13";
+
+  src = fetchFromGitHub {
+    owner = "issuu";
+    repo = pname;
+    rev = version;
+    hash =
+      "sha256-KFd43Ukz5gMeM3ik2VlfaIPwcZe9yaxk9VcQIrauUGU=";
+
+  };
+
+  nativeBuildInputs =
+    [ pkg-config protobuf ];
+  buildInputs = with ocaml-ng.ocamlPackages_4_14; [
+    zarith
+    ppx_deriving
+    ppx_deriving_yojson
+    re
+    dune-site
+    ppx_expect
+    protobuf
+  ];
+  doCheck = true;
+  nativeCheckInputs = [ protobuf ];
+
+  meta = {
+    homepage =
+      "https://github.com/issuu/ocaml-protoc-plugin";
+    description = "Maps google protobuf compiler to Ocaml types.";
+    longDescription = ''
+      The goal of Ocaml protoc plugin is to create an
+      up to date plugin for the google protobuf compiler
+      (protoc) to generate Ocaml types and serialization
+      and de-serialization function from a .proto file.
+    '';
+    license = lib.licenses.mit;
```

Comment:
license looks like apache https://github.com/issuu/ocaml-protoc-plugin/blob/master/LICENSE.md

Reply:
Indeed, changed it.